src/pages/PersonalData.js

```diff
@@ -0,0 +1,46 @@
+import styled from "styled-components";
+import { useNavigate } from "react-router-dom";
+
+export default function PersonalData({ saveTheData, inputData }) {
+  function savedInput(event) {
+    event.preventDefault();
+    const form = event.target;
+    const { name, email, address } = form.elements;
+    saveTheData(name.value, email.value, address.value);
+    /* console.log(name.value, email.value, address.value); */
+  }
+  return (
+    <>
+      <Section>
+        <form onSubmit={savedInput}>
+          <fieldset>
+            <legend>Gebe deine Persönliche Daten ein: </legend>
+            <label>Name: </label>
+            <input type="text" name="name" id="name"></input>
+            <label>Email: </label>
+            <input type="email" name="email" id="email"></input>
+            <label>Adresse: </label>
+            <input type="text" name="address" id="address"></input>
+            <button type="submit">Submit</button>
+          </fieldset>
+        </form>
+      </Section>
+    </>
```

Comment:
The `fraction` (`<>`) are not neccessary here. Because `<Section<` is already the one wrapping element.

Reply:
already done